rust/parquet/src/basic.rs

```diff
@@ -405,29 +405,29 @@ impl fmt::Display for ColumnOrder {
 impl convert::From<parquet::Type> for Type {
     fn from(value: parquet::Type) -> Self {
         match value {
-            parquet::Type::BOOLEAN => Type::BOOLEAN,
-            parquet::Type::INT32 => Type::INT32,
-            parquet::Type::INT64 => Type::INT64,
-            parquet::Type::INT96 => Type::INT96,
-            parquet::Type::FLOAT => Type::FLOAT,
-            parquet::Type::DOUBLE => Type::DOUBLE,
-            parquet::Type::BYTE_ARRAY => Type::BYTE_ARRAY,
-            parquet::Type::FIXED_LEN_BYTE_ARRAY => Type::FIXED_LEN_BYTE_ARRAY,
+            parquet::Type::Boolean => Type::BOOLEAN,
```

Comment:
@kornholi @liurenjie1024 @sunchao do you think it would be disruptive to change the local enums like `Type` and `LogicalType` to have camel casing? I'm assuming we've had them as upper case because earlier versions of `thrift` generated them like that?

Reply:
Arrow's enum has camel casing, so I think it may look better to change parquet's local enums to have camel casing.

Reply:
I'm somewhat confused why these duplicate enums exist in the first place. If we're going to depend on a specific version of `parquet-format`, why not use the types from there directly?

Reply:
Good question. I'll have a look to see why, but my answer won't be authoritative as @sunchao and @liurenjie1024 are most familiar with the parquet codebase.

Reply:
I'm not familiar with this part, but I guess this is used for better mapping of definitions of parquet.

Reply:
I think originally this was to give an isolation between the Thrift definitions and the definitions in parquet-rs, and also allow us to rename things a little differently.

Reply:
Yes the Thrift was using all upper cases before and changed because of the version bump. We may break dependencies if we change the cases for the local enums. Need to be careful with that. I'd say do that in a separate PR later.